lib/creek/styles/converter.rb

```diff
@@ -45,16 +46,18 @@ def self.call(value, type, style, options = {})
         when 'b'
           value.to_i == 1
         when 'str'
-          value
+          convert_html_entities(value)
```

Comment:
I understand why it is needed for type 'inlineStr'. But why why do we need to decode from html, even if the type is 'str'?